src/link.js

```diff
@@ -21,6 +21,50 @@ const getProjectDependencies = () => {
   return Object.keys(pjson.dependencies).filter(name => name !== 'react-native');
 };
 
+const makeLink = (project, dependency) => (cb) => {
+  if (project.android && dependency.config.android) {
+    log.info(`Linking ${dependency.name} android dependency`);
+
+    const didLink = registerDependencyAndroid(
+      dependency.name,
+      dependency.config.android,
+      project.android
+    );
+
+    if (didLink) {
+      log.info(`Android module ${packageName} has been successfully linked`);
```

Comment:
if you notify a user in a happy scenario, you should do the same for the opposite one.